### PR TITLE
docs(python): add polars-distance to plugins page

### PR DESCRIPTION
**Pull request by ion-elgreco**
Adding a link to the distance plugin, if that's ok : )

### CODE REVIEW

File: docs/user-guide/expressions/plugins.md [hunk 1]
```diff
@@ -249,5 +249,6 @@ That's all you need to know to get started. Take a look at this [repo](https://g
 Here is a curated (non-exhaustive) list of community implemented plugins.
 
 - [polars-business](https://github.com/MarcoGorelli/polars-business) Polars extension offering utilities for business day operations
+- [polars-distance](https://github.com/ion-elgreco/polars-distance) Polars plugin for pair-wise distance functions
```
Comment (MarcoGorelli) [inline docs/user-guide/expressions/plugins.md:252]: nitpick but I think pairwise is one word? https://www.oed.com/dictionary/pairwise_adv?tab=factsheet#32486661

Comment (ion-elgreco) [reply docs/user-guide/expressions/plugins.md]: Oops :P Fixed now

**Comment by ritchie46** [comment]
Seems good to me. I do want to make distinction between plugins that are reviewed on code by polars-core-developers and community. As that has influence on the performance you may expect of them.

**Comment by ion-elgreco** [comment]
@ritchie46 I understand, it mentions in the preface that it's community plugins though, but perhaps could be more emphasized.

I do however do my best to make them performant :P

**Comment by ion-elgreco** [comment]
@ritchie46 also just a thought, but maybe one of the core devs could take a glance to some of the plugins to give them a Polars Approved badge 🤔

**Comment by ritchie46** [comment]
Yes, I think we should try to get some reviewing in those directions, only atm moment the issue/PR tracker here is already exploding. :sweat_smile:

**Review by ritchie46 — APPROVED** [review]

**Comment by ion-elgreco** [comment]
@ritchie46 I understand it's growing alot lately :)

**Pull request merged**